code/modules/reagents/reagent_containers/food/drinks.dm

```diff
@@ -819,6 +821,8 @@
 	name = "Nuka Cola"
 	desc = "Cool, refreshing, Nuka Cola."
 	icon_state = "nuka"
+	tabself = "You pull back the cap on"
```

Comment:
Ino is correct, this language doesn't sound natural. Caps are unscrewed or popped off, not pulled back like tabs.

Reply:
would "pop off the cap" work?

Reply:
"Pop the cap off" would do